core/src/main/java/org/wildfly/extras/creaper/core/online/ExpectedFailuresBlock.java

```diff
@@ -0,0 +1,29 @@
+package org.wildfly.extras.creaper.core.online;
+
+import java.io.Closeable;
+
+/**
+ * <p>The only function this interface exposes is the ability to end a block of code started by
+ * {@link OnlineManagementClient#expectedFailures()}:</p>
+ *
+ * <pre>
+ * ExpectedFailuresBlock expectedFailures = client.expectedFailures();
+ * try {
+ *     ModelNodeResult result = client.execute(...);
+ *     ...
+ * } finally {
+ *     expectedFailures.close();
+ * }
+ * </pre>
+ *
+ * <p>If using Java 7 and above, a try-with-resources block will make the code shorter:</p>
+ *
+ * <pre>
+ * try (ExpectedFailuresBlock expectedFailures = client.expectedFailures()) {
+ *     ModelNodeResult result = client.execute(...);
+ *     ...
+ * }
+ * </pre>
+ */
+public interface ExpectedFailuresBlock extends Closeable {
+}
```

Comment:
Yea, I'm not particularly fond of the name. It's really meant for a very specific situation. I chose `ExpectedFailures` because outside of such blocks, failures are _not expected_, but I agree that a there should be a better name.